scripts/exglobal_extractvars.sh

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env bash                                                                                                                                                                    
+################################################################################
+#   Script:    
+#
+source "${USHgfs}/preamble.sh"
+
+# Programs used
+export WGRIB2=${WGRIB2:-${wgrib2_ROOT}/bin/wgrib2}
+
+# Scripts used
+EXTRCTVARA=${EXTRCTVARA:-"${USHgfs}/atmos_extractvars.sh"}
+EXTRCTVARO=${EXTRCTVARO:-"${USHgfs}/ocnice_extractvars.sh"}
+EXTRCTVARW=${EXTRCTVARW:-"${USHgfs}/wav_extractvars.sh"}
```

Comment:
Is there a reason we need pass-throughs here?
```suggestion
EXTRACT_ATMOS="${USHgfs}/atmos_extractvars.sh"
EXTRACT_MARINE="${USHgfs}/ocnice_extractvars.sh"
EXTRACT_WAVE="${USHgfs}/wav_extractvars.sh"
```